README.md

```diff
@@ -2,8 +2,8 @@
 
 [![Go Doc](https://godoc.org/github.com/stmcginnis/gofish?status.svg)](http://godoc.org/github.com/stmcginnis/gofish)
 [![Go Report Card](https://goreportcard.com/badge/github.com/stmcginnis/gofish?branch=main)](https://goreportcard.com/report/github.com/stmcginnis/gofish)
-[![Releases](https://img.shields.io/github/release/stmcginnis/gofish/all.svg?style=flat-square)](https://github.com/stmcginnis/gofish/releases)
-[![LICENSE](https://img.shields.io/github/license/stmcginnis/gofish.svg?style=flat-square)](https://github.com/stmcginnis/gofish/blob/main/LICENSE)
+[![Releases](https://img.shields.io/github/release/stmcginnis/gofish/all.svg?style=flat-square)](https://github.com/kozl/gofish/releases)
+[![LICENSE](https://img.shields.io/github/license/stmcginnis/gofish.svg?style=flat-square)](https://github.com/kozl/gofish/blob/main/LICENSE)
```

Comment:
I can enable the workflow to run, but first I think you made a bunch of these changes by accident. I usually just use a replace directive if I'm testing out local changes to a library so there's less mucking with the real source code.

There are a few lint issues from the earlier linting check, but doesn't look like it kept those comments around after the updates.

Reply:
Fixed this